content/en/docs/Guides/backup/_index.md

```diff
@@ -191,11 +234,12 @@ To initiate an automatic restore from a backup in an [S3 bucket](https://aws.ama
 	"sub": {
 		"access_key_id": "$ACCESS_KEY_ID",
 		"secret_access_key": "$SECRET_ACCESS_KEY_ID",
-		"endpoint": "$ENDPOINT",
 		"region": "$BUCKET_REGION",
 		"bucket": "$BUCKET_NAME",
 		"path": "backups/db.sqlite3.gz"
 	}
 }
 ```
 By default rqlite will exit with an error if it fails to download the backup file. If you wish an rqlite node to continue starting up even if the download fails, set `continue_on_failure: true`.
+
+Note that the `sub` configuration examples for non-Amazon S3 storage (such was Wasabi and MinIO) from the Automated Backups section above apply here as well.
```

Comment:
Add a section header here, so this important point stands out. E.g.

#### Other S3-compliant providers
The `sub` configuration examples for non-Amazon S3 storage from the _Automated Backups_ section above apply equally well to _Automatic Restores_,. This allows you to download a previously uploaded backup from, for example, Wasabi and MinIO.

Reply:
> Other S3-compliant providers

Should I use the subheading "Non-Amazon S3 Storage" instead so that it aligns with the backup section, or rename the subheading in the backup section to "Other S3-compliant providers"?

Reply:
I left them as proposed (different) so you can try it on for size.